protocol/persistence_test.go

```diff
@@ -2035,18 +2035,64 @@ func TestGetCommunityMemberAllNonDeletedMessages(t *testing.T) {
 	messages, err = p.GetCommunityMemberAllMessagesID(testPK, testCommunity)
 	require.NoError(t, err)
 	require.Len(t, messages, 1)
-	require.Exactly(t, "1", messages[0].ID)
-	require.Exactly(t, testPublicChatID, messages[0].ChatID)
+	require.Exactly(t, "1", messages[0].Id)
+	require.Exactly(t, testPublicChatID, messages[0].ChatId)
```

Comment:
Almost everywhere else in the code, we use `ID` with capital letters. Why does this one have a lower `d`?